### PR TITLE
Fix owners guide: approver in the reviewers section

### PR DESCRIPTION
**Pull request by xinydev**
As the guide says: 

> Good examples of OWNERS usage:
>   the approvers are not in the reviewers section

**Which issue(s) this PR fixes**:

Fixes #

### CODE REVIEW

File: contributors/guide/owners.md [hunk 1]
```diff
@@ -53,8 +53,8 @@ approvers:
   - alice
   - bob     # this is a comment
 reviewers:
-  - alice
-  - carol   # this is another comment
```
[the remark this answers is not in the document]
Comment (xinydev) [reply contributors/guide/owners.md:57]: https://github.com/kubernetes/community/blob/master/contributors/guide/owners.md#maintaining-owners-files. 

> Good examples of OWNERS usage:
> - team aliases are used that correspond to sigs
> - there are more `reviewers` than `approvers`
> - the `approvers` are not in the `reviewers` section 
> - OWNERS files that are regularly updated (at least once per release)

Maybe we need to delete `the approvers are not in the reviewers section`? I do see  a lot of people in both groups

[the remark this answers is not in the document]
Comment (mrbobbytables) [reply contributors/guide/owners.md:57]: That might be a good idea, generally the IDEA is to have the reviewer pool be a different set of people but its not often the case.

[the remark this answers is not in the document]
Comment (xinydev) [reply contributors/guide/owners.md:57]: I'm not sure. Or `the approvers are not in the reviewers section` is the ideal state, but we haven't reached it yet?

> This goes against the idea of having at least two sets of eyes on a PR, and may be a sign that there are too few reviewers

[the remark this answers is not in the document]
Comment (mrbobbytables) [reply contributors/guide/owners.md:57]: Actually, look at the text above its for a Good example where it is the ideal state.
Sorry for the back and forth on this - I'll go ahead and approve your change. 

/approve
/lgtm

**Pull request merged**